shortstories/models.py

```diff
@@ -0,0 +1,12 @@
+from django.db import models
+class ShortStory(models.Model):
+     status=(
+        ('dr', 'Draft'),
+        ('p' , 'Published'),
+        ('b' , 'banned'),
+        ('de', 'deleted'),
+    )
+    author =models.Foreignkey(user,on_delete=models.CASCADE)
+    title = models.CharField(max_length=500)
+    body = models.CharField(min_length=3500,Choices = status)
+    publication_date = models.DateTimeField('date published')
```

Comment:
I don't think 'date published' is a correct syntax in your publication date, you'll probably see error for this